CustomScripts/hide-all-floats.sh

```
@@ -1,34 +1,42 @@
 #!/usr/bin/env bash
 
+TEMP_FILE="/tmp/minimized_windows.txt"
+
 # Get the list of floating windows in the current space
-floating_windows=($(yabai -m query --windows --space | jq '.[] | select(.["is-floating"] == true) | .id'))
+floating_windows=($(yabai -m query --windows --space | jq '.[] | select((.["is-floating"] == true) and (.["is-visible"] == true)) | select(.subrole == "AXStandardWindow") | .id'))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)